products/harbor.md

```diff
@@ -0,0 +1,95 @@
+---
+title: VMware Harbor Registry
+category: server-app
+tags: cncf
+iconSlug: harbor
+permalink: /harbor
+changelogTemplate: https://github.com/goharbor/harbor/releases/tag/v__LATEST__
+releaseDateColumn: true
+eolColumn: End of general support
+
+auto:
+-   git: https://github.com/goharbor/harbor.git
+
+releases:
+-   releaseCycle: "2.9"
+    releaseDate: 2023-09-15
+    eol: 2024-09-30
+    latest: "2.9.1"
+    latestReleaseDate: 2023-11-28
+
+-   releaseCycle: "2.8"
+    releaseDate: 2023-05-22
+    eol: 2024-05-31
+    latest: "2.8.4"
+    latestReleaseDate: 2023-09-14
+
+-   releaseCycle: "2.7"
+    releaseDate: 2023-01-05
+    eol: 2024-01-31
+    latest: "2.7.4"
+    latestReleaseDate: 2023-11-30
+
+-   releaseCycle: "2.6"
+    releaseDate: 2022-09-01
+    eol: 2023-06-30
+    latest: "2.6.4"
+    latestReleaseDate: 2023-03-14
+
+-   releaseCycle: "2.5"
+    releaseDate: 2022-04-26
+    eol: 2023-01-31
+    latest: "2.5.4"
+    latestReleaseDate: 2022-09-01
+
+-   releaseCycle: "2.4"
+    releaseDate: 2021-11-03
+    eol: 2022-11-30
+    latest: "2.4.3"
+    latestReleaseDate: 2022-08-18
+
+-   releaseCycle: "2.3"
+    releaseDate: 2021-08-16
+    eol: 2022-05-31
+    latest: "2.3.5"
+    latestReleaseDate: 2021-12-18
+
+-   releaseCycle: "2.2"
+    releaseDate: 2021-04-19
+    eol: 2022-04-19
+    latest: "2.2.3"
+    latestReleaseDate: 2021-07-15
+
+-   releaseCycle: "2.1"
+    releaseDate: 2020-10-23
+    eol: 2021-10-23
+    latest: "2.1.6"
+    latestReleaseDate: 2021-07-19
+
+-   releaseCycle: "2.0"
+    releaseDate: 2020-07-09
+    eol: 2021-07-02
+    latest: "2.0.6"
+    latestReleaseDate: 2021-03-20
+
+-   releaseCycle: "1.10"
+    releaseDate: 2019-12-19
+    eol: 2020-11-18
+    latest: "1.10.3"
+    latestReleaseDate: 2020-06-13
+
+-   releaseCycle: "1.9"
+    releaseDate: 2019-11-18
+    eol: 2020-11-18
+    latest: "1.9.4"
+    latestReleaseDate: 2020-01-08
+
+---
+
+> [Harbor](https://goharbor.io/) is an open source registry that secures artifacts with
+> policies and role-based access control, ensures images are scanned and free from
+> vulnerabilities, and signs images as trusted.
```

Comment:
This is about OpenSource Harbor and needs to be updated. Perhaps you can summarize from here: https://docs.vmware.com/en/VMware-Harbor-Registry/services/vmware-harbor-registry/GUID-index.html

Reply:
Done, thanks a lot for your in-depth review, seems like I did a lot of mistakes 😖